### PR TITLE
Updating deployers for apps using shared runtime

### PR DESCRIPTION
**Pull request by JunTaoLuo**
#718

### CODE REVIEW

**Comment by JunTaoLuo** [comment]
Oops I mean to merge this into release not dev

File: src/Microsoft.AspNetCore.Server.Testing/Common/DeploymentParameters.cs [hunk 1]
```diff
@@ -74,6 +74,8 @@ public class DeploymentParameters
         /// </summary>
         public bool PublishApplicationBeforeDeployment { get; set; }
 
+        public bool UseSharedRuntime { get; set; }
```
Comment (Tratcher) [inline src/Microsoft.AspNetCore.Server.Testing/Common/DeploymentParameters.cs]: Maybe an enum? It would better describe the else case.

Comment (Tratcher) [inline src/Microsoft.AspNetCore.Server.Testing/Common/DeploymentParameters.cs]: Default to shared? That's what we changed all of our samples to, correct?

**Comment by JunTaoLuo** [comment]
:up::date:

**Comment by Tratcher** [comment]
:shipit:

**Pull request merged**